src/screens/ProfileScreen/ProfileScreen2.tsx

```diff
@@ -29,7 +29,7 @@ const ActiveScreen = () => {
         const contributionResponse = await fetchContribution(userName);
         setActiveTasks(
           contributionResponse.all.filter(
-            (item) => item.task.status === 'ACTIVE',
+            (item) => item.task.status === 'IN_PROGRESS',
```

Comment:
please give both , ACTIVE and IN_PROGRESS , infact which is not completed , other than that everything should be in active tasks